### ISSUE TITLE
Lacking correct button type, aria-role, id, on login page

### ISSUE THREAD

**Issue by nikonikoniko** [aula-app/aula-frontend#563]
<img width="701" alt="Image" src="https://github.com/user-attachments/assets/364b0e4c-74b3-4e32-8852-ba9d52120dc7" />

button should be `type=submit`, or have a `submit` aria role
